{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "ed9ce887",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 0, 'page_label': '1'}, page_content='Analysis of CYP701A1 genes in gossypium species and functional\\ncharacterization through gene silencing\\nZhao Lianga, Di Jiachunb, Guo Qia, Xu Zhenzhena, Zhao Juna, Xu Penga, Xu Jianwena,\\nLiu Jianguanga, Shen Xinliana, Chen Xushenga,*\\na Key Laboratory of Cotton and Rapeseed (Nanjing), Ministry of Agriculture and Rural Affairs, Institute of Industrial Crops, Jiangsu Academy of Agricultural Sciences,\\nNanjing, China\\nb Institute of Germplasm Resources and Biotechnology, Jiangsu Academy of Agricultural Sciences, Nanjing, China\\nARTICLE INFO\\nKeywords:\\nCotton\\nCytochrome P450\\nGibberellins\\nCYP701\\nRegulatory network\\nABSTRACT\\nGibberellins (GA) are known to play crucial roles in various aspects of plant growth and development. The cy-\\ntochrome P450 enzyme family is recognized for its signiﬁcance in plant metabolic processes. Speciﬁcally,\\nCYP701s, a subgroup of CYP71, encodeent-kaurene oxidase in the gibberellin synthesis pathway. In this study, we\\nanalyzed genomic data from 30Gossypium species, including nine allotetraploid genomes (AD1-AD7, with two\\neach for AD1 and AD2), 21 diploid genomes (A-G, K, with two A-genomes and 12 D-genomes), andGossypioides\\nkirkii genome as an outgroup for evolutionary analysis, totaling 31 genomes. Subsequently, 40CYP701A1 genes\\nwere identiﬁed from various genomes and conducted a comprehensive analysis of their structure and evolution.\\nVirus-induced gene silencing (VIGS) technology was utilized to knock out theGhCYP701A1 gene in Gossypium\\nhirsutum ac TM-1. Subsequent analysis revealed changes in hormone content, with decreased gibberellin levels\\nand notable increases in auxin, cytokinin, and jasmonic acid contents. Conversely, salicylic acid content\\ndecreased, while the precursor for ethylene synthesis, 1-aminocyclopropane-1-carboxylic acid (ACC), remained\\nrelatively stable. Transcriptome analysis of the gene silencing plants identiﬁed 15,962 differentially expressed\\ngenes, including 8376 upregulated and 7586 downregulated genes. Enrichment analysis through KEGG pathway\\nhighlighted ‘Plant hormone signal transduction’ as a prominent pathway with 234 differentially expressed genes.\\nThe study provided insights into the function and regulatory network of the gene.\\n1. Introduction\\nGibberellins (GAs) play a crucial role in various aspects of plant\\ngrowth and development, such as seed germination, stem elongation, leaf\\nexpansion, and ﬂower and seed developments [1–3]. Despite the exis-\\ntence of numerous GAs identiﬁed from plants and fungi [4], only a select\\nfew, including GA1, GA3, GA4, and GA7 [5], exhibit biological activity.\\nThe majority of other gibberellins act as intermediates or catabolites in\\nthe biosynthesis of the active GAs. The biosynthesis of GAs in higher\\nplants involves three main stages: the production ofent-kaurene from\\ngeranyl geranyl diphosphate (GGDP) in proplastids, the conversion of\\nent-kaurene to GA12 through cytochrome P450 monooxygenases, and\\nthe formation of C20-GAs and C19-GAs in the cytoplasm. Various en-\\nzymes, such as ent-copalyl diphosphate synthase (CPS), ent-kaurene\\nsynthase (KS), ent-kaurene oxidase (KO), ent-kaurenoic acid oxidase\\n(KAO), and GA 20-oxidases (GA20ox) and 3-oxidases (GA3ox), catalyze\\nthese reactions [6]. The cytochrome P450 enzyme family, particularly\\nP450, play signiﬁcant roles in plant metabolic processes and serves as a\\nmodel for metabolic structure and evolution [7,8]. CYP701s, a subgroup\\nof the cytochrome P450 superfamily, are known for their involvement in\\nvarious metabolic pathways across different organisms. CYP701s have\\nbeen present since early in the evolution of land plants, with examples\\nfound in moss (Physcomitrella). Unlike most oxygenases in terpenoid\\nhormone biosynthesis, CYP701s are part of the CYP71 clan [9,10].\\nCYP701A3, for instance, encodes theent-kaurene oxidase (KO) respon-\\nsible for three oxygenation steps in the conversion ofent-kaurene to\\nent-kaurenoic acid, inﬂuencing processes likeﬂowering regulation, stem\\nelongation, and cell expansion [11–13].\\n* Corresponding author.\\nE-mail addresses: Liangz@jaas.ac.cn (Z. Liang), 1103156363@qq.com (D. Jiachun), sevenguo86@163.com (G. Qi), lele20032@163.com (X. Zhenzhen),\\nsxzhaojun88@aliyun.com (Z. Jun), semon528@hotmail.com (X. Peng), xujianwen@jaas.ac.cn (X. Jianwen), 22626451@qq.com (L. Jianguang), xlshen68@126.\\ncom (S. Xinlian), 19890003@jaas.ac.cn (C. Xusheng).\\nContents lists available atScienceDirect\\nCrop Design\\njournal homepage: www.journals.elsevier.com/crop-design\\nhttps://doi.org/10.1016/j.cropd.2024.100081\\nReceived 9 June 2024; Accepted 11 September 2024\\nAvailable online 16 September 2024\\n2772-8994/© 2024 The Authors. Published by Elsevier B.V. on behalf of Zhejiang University and Zhejiang University Press Co. Ltd. This is an open access article under\\nthe CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).\\nCrop Design 4 (2025) 100081'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 1, 'page_label': '2'}, page_content=\"Cotton, as one of the world's most importantﬁber crops, serves as the\\nraw material for textiles and plays a signiﬁcant role in global economies.\\nIn recent years, the completion of the cotton genome sequencing has\\nmarked a signiﬁcant milestone in agricultural genomics. The cotton\\ngenome sequence provides a detailed blueprint of the genetic composi-\\ntion of various cotton species, including the cultivated tetraploid cotton\\nspecies, Gossypium hirsutum and Gossypium barbadense, along with their\\ndiploid progenitors. Through genome sequencing, researchers have un-\\ncovered the intricate genetic landscape of cotton, characterized by\\npolyploidy, extensive gene duplication, and genome rearrangements [14,\\n15].\\nIn this study, we utilized genomic data from a total of 31 different\\nsources. This included nine tetraploid germplasm resources of cotton\\n(AD1-AD7) [16–19], with AD1 representingG. hirsutum acc. TM-1 and\\nthe early form of domesticated G. hirsutum known as race punctatum\\n(Ghp), and AD2 representingG. barbadense acc. H7124 and a perennial\\nsea-island cotton accession, Lianhemumian. Additionally, we analyzed\\n22 diploid genomes, which consisted of eight genomes from the A-K\\ngroup (A genome containing A1 and A2, D genome containing D1-D10, G\\ngenome containing G1 and G2) [14,15,20–27], as well as genome in-\\nformation from the exoid groupGossypioides kirkii[28]. By comparing the\\nsequence information of GhCYP701A1 in TM-1 with homologous se-\\nquences from the other genomes, we conducted an evolutionary analysis.\\nFurthermore, we utilized virus-induced gene silencing (VIGS) knockout\\nand transcriptome sequencing to investigate the function and regulatory\\nnetwork of theGhCYP701A1 gene.\\n2. Materials and methods\\n2.1. Plant material\\nGossypium hirsutum cv. TM-1 plants were cultivated in a controlled\\ncotton climatic chamber at a temperature of 30/25/C6 3 /C14 C with a light/\\ndark cycle of 16/8 h. After seven days from sowing, the cotyledons of\\nseedlings were targeted for virus-mediated silencing. Concurrently, the\\nenvironmental conditions were adjusted to 8 h of darkness and 16 h of\\nlight at 23/C14 C. Following this, the leaves of the plants were harvested and\\nimmediately frozen in liquid nitrogen. All collected materials were then\\nstored at/C0 80 /C14 C until further processing. The three biological replicates\\nwere split into two portions, with one used for RNA isolation and Illu-\\nmina sequencing, while the other was utilized for hormone\\nmeasurements.\\n2.2. Identiﬁcation and structural analysis of CYP701A1\\nThe exon/intron structures of theCYP701A1 genes were visualized in\\nvarious genomes using the Gene Structure Display Server [29]. The\\nstructural domains and motifs of the predicted proteins were validated\\nthrough Batch CD-Search (https://www.ncbi.nlm.nih.gov/Structure/b\\nwrpsb/bwrpsb.cgi). Subsequently, a neighbor-likelihood (NL) phyloge-\\nnetic tree of all identiﬁed CYP701A1 genes was generated using MEGA-X\\nwith 1000 bootstrap replications [30]. The promoter sequence (2000bp\\nDNA sequence upstream of the start codon) of eachCYP701A1 gene was\\nextracted from the genomes and analyzed forcis-acting regulatory ele-\\nments using the PlantCARE database [31]. The predictedcis-acting reg-\\nulatory elements were then classiﬁed based on their regulatory functions.\\n2.3. Validation of gene expression proﬁle using qRT-PCR\\nRNA was extracted from samples for analysis using real-time quan-\\ntitative (qRT-PCR). cDNA synthesis was performed using M-MLV Reverse\\nTranscriptase (Promega, U.S). qRT-PCR speciﬁc primers were designed\\nwith Beacon Designer 7.0 software from Premier Biosoft International,\\nPalo Alto, CA. The internal control used wasHistone3 (AF024716). qRT-\\nPCR was conducted using iQ SYBR Green Supremix (Bio-Rad, USA)\\nfollowing the manufacturer's instructions. The PCR thermal cycle\\nconditions were as follows: 94/C14 C for 3 min, 40 cycles including 94/C14 C for\\n15 s, 60/C14 C for 15 s, and 72/C14 C for 30 s. The relative expression levels were\\ndetermined using the 2/C0 ΔΔCt method [32].\\n2.4. The experimental method for virus-induced gene silencing (VIGS)\\nVirus-Induced Gene Silencing (VIGS) was employed in this research\\nto investigate gene function. The VIGS vector was constructed based on\\nthe Tobacco Rattle Virus (TRV) system. Initially, a segment of the target\\ngene was ampliﬁed through PCR and inserted into the TRV vector using\\nspeciﬁc restriction enzymes. The resulting vector was then introduced\\ninto Agrobacterium tumefaciens strain GV3101, which was subsequently\\nused to inﬁltrate cotton cotyledon. The inﬁltrated plants were grown\\nunder controlled conditions with a 16-h light/8-h dark photoperiod at 23\\n/C14 C. Gene silencing was conﬁrmed through quantitative reverse tran-\\nscription PCR (qRT-PCR) analysis to assess the reduction in target gene\\nexpression compared to control plants inﬁltrated with an empty vector\\n[33].\\n2.5. Plant hormone liquid chromatography-mass spectrometry (LC-MS)\\ndetection\\nThis experiment employed the AB Company's Qtrap6500 mass spec-\\ntrometer. The ESI-HPLC-MS/MS method was utilized for quantitative\\nanalysis of plant hormones. The Agilent 1290 high-performance liquid\\nchromatography (HPLC) system was used to separate sample substances,\\nwith electrospray ionization (ESI) as the ion source, and multiple reaction\\nmonitoring (MRM) mode scanning was performed. This involved\\nsimultaneous scanning of multiple selected ion monitoring (SIM) modes.\\nThe peak time of the target substance in the sample and standard, as well\\nas the ratio of the target substance's various daughter ions in the sample\\nand standard, were used to determine the analyte in the sample. Quan-\\ntitation of the analyte was achieved using the standard curve method.\\nApproximately 1 g of fresh plant samples were accurately weighed and\\nground into powder in liquid nitrogen. The powder was mixed with 10 ml\\nof isopropanol/hydrochloric acid extraction buffer and shaken at 4/C14 C for\\n30 min. Then, 20 ml of dichloromethane was added, followed by shaking\\nat 4/C14 C for another 30 min. After centrifugation at 4/C14 C and 13,000 rpm\\nfor 5 min, the organic phase was collected. The organic phase was dried\\nunder nitrogen and dissolved in 400μl of methanol (0.1 % formic acid).\\nThe solution wasﬁltered through a 0.22μm membrane and subjected to\\nHPLC-MS/MS analysis. Standard solutions of indole-3-Acetic acid (IAA),\\n1-aminocyclopropane-1-carboxylic acid (ACC), Gibberellin (GA3), jas-\\nmonic acid (JA), salicylic acid (SA), abscisic acid (ABA) were prepared in\\nmethanol (0.1 % formic acid) with concentrations 0.1 ng/ml,0.2 ng/\\nml,0.5 ng/ml,1 ng/ml,2 ng/ml,5 ng/ml, 10 ng/ml, 20 ng/ml, 50 ng/ml\\n200 ng/ml. Each concentration was prepared in triplicate, and points\\nwith poor linearity were removed when constructing the standard curve\\nequation. All experiments were conducted at least three times per\\nsample.\\n2.6. Transcriptome sequencing and DEG analysis\\nTranscriptome analysis of differential gene expression was performed\\nusing a combination of experimental and computational techniques. RNA\\nwas extracted from control and experimental samples representing\\ndifferent conditions or treatments. The RNA quality and purity were\\nassessed by using a Qubit® 2.0 Fluorometer (Invitrogen, Carlsbad, CA,\\nU.S.) and an Agilent 2100 Bioanalyzer (Agilent Technologies, U.S.).\\nSubsequently, high-throughput RNA sequencing (RNA-seq) was con-\\nducted using Illumina sequencing platforms to generate transcriptomic\\ndata.\\nRaw sequencing reads were subjected to quality control using FastQC\\n(https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)t o\\nassess read quality and remove low-quality reads. Adaptor sequences and\\nlow-quality bases were trimmed using Trimmomatic (http://www.us\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n2\"),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 2, 'page_label': '3'}, page_content='Table 1\\nBasic information about theCYP701A1 gene from different genomes.\\nGenes Species Genome Chr Location NO.\\nCDS\\nNO.\\naa\\nMW\\n(KDa)\\nTheoretical\\npI\\nInstability\\nIndex\\nAliphatic\\nIndex\\nGrand Average of\\nHydropathicityFrom To\\nGheCYP701A1 G. herbaceum A1 Chr06 1,483,073 1,486,148 1509 502 57.68 7.65 44.27 87.23 /C0 0.324\\nGarCYP701A1 G. arboreum A2 Chr06 1,466,718 1,469,742 1509 502 57.76 8.28 44.2 88.01 /C0 0.319\\nGanoCYP701A1 G. anomalum B1 Chr06 1,625,253 1,628,333 1509 502 57.70 8.03 44.25 87.43 /C0 0.309\\nGstuCYP701A1 G. sturtianum C1 Chr06 161,707,714 161,710,978 1509 502 57.62 8.29 43.43 87.81 /C0 0.314\\nGthCYP701A1 G. thurberi D1 Chr10 1,432,896 1,435,965 1509 502 57.57 8.04 43.56 87.43 /C0 0.31\\nGkarCYP701A1 G. harknessii D2 Chr06 1,140,040 1,142,524 1515 504 57.90 8.28 45.42 87.28 /C0 0.313\\nGdavCYP701A1 G. davidsonii D3 Chr10 1,357,044 1,359,967 1509 502 57.65 8.03 45.32 87.43 /C0 0.323\\nGkloCYP701A1 G. klotzschianum D3 Chr06 1,483,076 1,485,999 1509 502 57.65 8.03 45.32 87.43 /C0 0.323\\nGaridCYP701A1 G. aridum D4 CM025189.1 979,772 982,726 1527 508 58.08 7.65 44.85 87.17 /C0 0.293\\nGraCYP701A1 G. raimondii D5 Chr10 1,221,824 1,224,656 1509 502 57.65 8.03 45.83 88.21 /C0 0.318\\nGgosCYP701A1 G. gossypioides D6 Chr06 1,253,575 1,256,523 1515 504 57.88 8.03 44.91 88.63 /C0 0.294\\nGlobCYP701A1 G. lobatum D7 CM025280.1 1,127,532 1,130,501 1521 506 57.82 8.03 44.48 88.08 /C0 0.287\\nGtriCYP701A1 G. trilobum D8 Chr06 1,232,260 1,235,338 1509 502 57.68 8.03 43.69 87.63 /C0 0.318\\nGlaxCYP701A1 G. laxum D9 CM025267.1 1,483,173 1,484,701 1221 406 47.13 6.26 45.14 86.26 /C0 0.418\\nGtuCYP701A1 G. turneri D10 D10_06 59,911,865 59,914,250 1416 471 54.18 6.96 43.55 85.94 /C0 0.406\\nGschCYP701A1 G. schwendimanii D11 CM025254.1 970,217 973,165 1515 504 57.70 7.65 44.21 85.73 /C0 0.319\\nGstoCYP701A1 G. stocksii E1 Chr06 2,052,726 2,055,852 1509 502 57.65 7.65 45.48 87.43 /C0 0.3\\nGloCYP701A1 G. longicalyx F1 Chr06 1,328,587 1,331,641 1509 502 57.64 7.65 44.89 87.05 /C0 0.323\\nGbiCYP701A1 G. bickii G1 chr06 132,156,670 132,159,784 1611 536 61.62 7.95 44.65 88.6 /C0 0.228\\nGauCYP701A1 G. australe G2 CM016619.1 144,183,953 144,187,074 1467 488 56.63 7.65 39.49 82.77 /C0 0.396\\nGrotCYP701A1 G. rotundifolium K2 Chr06 201,792,757 201,795,797 1509 502 57.78 8.47 44.6 89.36 /C0 0.325\\nGkiCYP701A1 Gossypioides kirkii – KI_06 1,296,432 1,298,459 1407 468 53.77 8.02 39.85 86.47 /C0 0.418\\nGhpCYP701A1_Dt G. hirsutum, race punctatum AD1 CM045517.1 1,491,758 1,494,096 1080 359 41.37 9.31 41.6 92.9 /C0 0.14\\nGhlCYP701A1_Dt G. barbadense, race\\nLianhemumian\\nAD2 D06 1,517,935 1,520,866 1515 504 57.83 8.03 44.41 88.43 /C0 0.3\\nGtCYP701A1_Dt G. tomentosum AD3 D06 1,432,204 1,435,145 1515 504 57.80 8.03 44.79 88.06 /C0 0.303\\nGmCYP701A1_Dt G. mustelinum AD4 CM017654.1 1,497,639 1,499,091 1101 366 42.57 5.71 46.72 87.43 /C0 0.437\\nGdCYP701A1_Dt G. darwinii AD5 CM017706.1 1,437,974 1,440,905 1515 504 57.82 8.03 45.93 87.86 /C0 0.31\\nGeCYP701A1_Dt G. ekmanianum AD6 CM046642.1 1,580,201 1,583,002 1473 490 56.28 9.46 44.29 86.35 /C0 0.271\\nGsCYP701A1_Dt G. stephensii AD7 CM045536.1 1,475,136 1,478,080 1521 506 57.99 8.03 45.04 88.28 /C0 0.297\\nGhpCYP701A1_At G. hirsutum, race punctatum AD1 CM045504.1 1,518,060 1,521,115 1509 502 57.73 8.03 44.67 87.43 /C0 0.337\\nGhlCYP701A1_At G. barbadense, race\\nLianhemumian\\nAD2 A06 1,553,313 1,556,359 1509 502 57.73 8.03 42.81 87.81 /C0 0.326\\nGtCYP701A1_At G. tomentosum AD3 A06 1,501,754 1,504,682 1416 471 54.23 6.62 42.53 85.31 /C0 0.432\\nGmCYP701A1_At G. mustelinum AD4 CM017641.1 1,472,398 1,475,455 1509 502 57.72 8.03 44.16 86.65 /C0 0.338\\nGdCYP701A1_At G . darwinii AD5 CM017693.1 1,436,147 1,439,194 1509 502 57.72 8.03 42.7 87.81 /C0 0.322\\nGeCYP701A1_At G. ekmanianum AD6 CM046641.1 1,444,555 1,447,474 1467 488 56.18 9.42 42.93 88.09 /C0 0.253\\nGsCYP701A1_At G. stephensii AD7 CM045535.1 1,574,370 1,577,426 1509 502 57.72 8.03 43.68 87.23 /C0 0.339\\nGbCYP701A1_Dt G. barbadense AD2 D06 1,405,252 1,408,184 1515 504 57.82 8.03 45.93 87.86 /C0 0.31\\nGbCYP701A1_At G. barbadense AD2 A06 1,428,320 1,431,365 1509 502 57.73 8.03 42.81 87.81 /C0 0.326\\nGhCYP701A1_Dt G. hirsutum AD1 D06 1,318,289 1,321,227 1515 504 57.82 8.03 45.18 87.86 /C0 0.31\\nGhCYP701A1_At G. hirsutum AD1 A06 1,394,845 1,397,900 1509 502 57.73 8.03 44.67 87.43 /C0 0.337\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n3'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 3, 'page_label': '4'}, page_content='adellab.org/cms/?page&equals;trimmomatic).\\nCleaned reads were then aligned to a reference genome or tran-\\nscriptome using alignment tools Bowtie2 (http://bowtie-bio.sourceforge\\n.net/bowtie2/index.shtml). The aligned reads were quantiﬁed to esti-\\nmate gene expression levels using tools HTSeq (https://htseq.readthe\\ndocs.io/en/master/).\\nTo identify differentially expressed genes (DEGs), we adopted a\\nconservative criteria by selection of the consistent results of cuffdiff (ref)\\nwith log2\\n(fold change) /C21 1o r /C20 -1 and also being signiﬁcantly expressed\\nwith FDR corrections at the level of 0.05 and genes FPKM value/C21 1.\\n3. Results\\n3.1. Sequence analysis of CYP701A1 from different gossypium genome\\nPhylogenetic analysis of 679 P450 family members in the TM-1\\ngenome and 237 members of Arabidopsis thaliana revealed that\\nGh_A05G1711, Gh_D05G1901, Gh_A06G0128, and Gh_D06G0106 genes\\nare grouped together with AtCYP701A3 (Fig. S1). Particularly, the\\nexpression level ofGh_D06G0106 in the transcriptome data of TM-1 tis-\\nsue expression was notably higher than other genes (Fig. S2), leading to a\\nfocus on Gh_A06G0128 and Gh_D06G0106 genes, which were subse-\\nquently namedGhCYP701A1. Through homologous comparison, a total\\nof 40 CYP701A1 genes were obtained from different cotton genomes,\\nincluding 22 from different diploid genomes and 18 from heterologous\\ntetraploids, with nine each from the At and Dt sub-genomes. The smallest\\nprotein expressed by these genes consists of 359 amino acids, originating\\nfrom the Dt sub-genome of upland cotton, an early domesticated species -\\nGossypium hirsutum var. Punctatum, known as GhpCYP701A1_Dt, while\\nthe largest protein comprises 536 amino acids from the G1 genome of\\nGossypium bickii, GbiCYP701A1. The molecular weights of the proteins\\nrange from 41.37 kDa to 61.62 kDa, with theoretical isoelectric points\\nbetween 5.71 and 9.46. Additionally, all proteins are hydrophobic in\\nnature (Table 1).\\nUsing NCBI, it was predicted that theseCYP701 genes from different\\ngenomes all contain a conserved P450 superfamily domain consisting of\\n259–465 amino acids (Fig. 1A). Correspondingly, these 40 genes have at\\nleast four introns interspersed within the coding sequences (GhpCY-\\nP701A1_Dt, GmCYP701_Dt and GlaxCYP701A1), with two genes,GeCY-\\nP701A1_At and GeCYP701A1_Dt, from the At and Dt subgenomes of\\nGossypium ekmanianum of the AD6 genomes containing ﬁve introns,\\nwhile the remaining genes contain 6 introns each (Fig. 1B).\\n3.2. Phylogenetic analysis of CYP701A1 from different gossypium genome\\nTo determine the evolutionary relationships between 40CYP701A1\\ngenes from different genomes, a maximum likelihood method was\\nemployed to construct their evolutionary tree, withGossypioides kirkii\\nused as an outgroup for tree construction. The results indicate that\\nGanoCYP701A1 from the B1 genome is evolutionarily closest toGki-\\nCYP701A1 from the outgroup Gossypioides kirkii . The remaining\\nCYP701A1 genes from different genomes can be classiﬁed into three\\nmajor clades: theﬁrst clade includesCYP701A1 genes from the heter-\\nologous tetraploid At sub-genome, along with those from the A1, A2, and\\nF1 genomes; the second clade comprisesCYP701A1 genes from the C1,\\nE1, G1, G2, and K genomes; and the third clade consists ofCYP701A1\\ngenes from different D genomes and the heterologous tetraploid Dt sub-\\ngenome. Regarding the early domesticated species,Gossypium hirsutum\\nvar. Punctatum and Gossypium barbadense var Lianhemumian, the\\nCYP701A1 genes from the At sub-genome exhibit the closest phyloge-\\nnetic relationship in both early domesticated and cultivated species,\\nwhereas this is not observed for the Dt sub-genome (Fig. 1B).\\nFig. 1. Identiﬁcation and bioinformatics analysis ofCYP701A1 from different cotton genomes.\\nA. Conserved domain; B. Evolution and gene structure analysis ofCYP701A1 gene from different genomes; C. Promotercis-acting element analysis.\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n4'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 4, 'page_label': '5'}, page_content='3.3. Promoter analysis of CYP701A1 from different gossypium genome\\nThe upstream 2000-bp sequences of each of these 40 genes in their\\nrespective genomes were extracted for promoter analysis. Through\\nanalysis, it was found that, excluding the core promoter elements (TATA-\\nbox and CAAT-box), a total of 15 different types of cis-regulatory ele-\\nments were identiﬁed. Among them, the most abundant elements were\\nlight-responsive elements, such as G-BOX, GT1-motif, GA-motif, GATA-\\nmotif, etc. (Fig. 1C).\\n3.4. The function of GhCYP701A1 was analyzed through gene silencing\\nTo verify the function ofGhCYP701A1, we employed VIGS technol-\\nogy to silence this gene inG. hirsutum ac. TM-1. The 553bp sequence from\\nGH_A06G0147 was cloned into the pTRV2 vector to suppress the\\nexpression of this gene in plant bodies. Upon the appearance of albino\\nphenotype in the plants, the expression level ofGhCYP701A1 gene in\\nsilenced plants was assessed, and the plant height was measured when\\nthe plants grew to four true leaves. The results revealed a signiﬁcant\\ndecrease in the expression level of theGhCYP701A1 gene in silenced\\nplants, accompanied by a markedly lower plant height compared to\\ncontrol plants (Fig. 2A–C). Subsequently, hormone content was deter-\\nmined in silenced and control plants. The results showed a signiﬁcant\\nreduction in gibberellin GA3 content in gene-silenced plants compared to\\ncontrol plants (Fig. 2D), while the content of auxin, zeatin, and methyl\\njasmonate was signiﬁcantly increased in silenced plants (Fig. 2E–G);\\nthere was no signiﬁcant difference in the content of ethylene precursor\\nACC between silenced plants and controls (Fig. 2H), although the content\\nof salicylic acid was signiﬁcantly lower in silenced plants compared to\\ncontrols (Fig. 2I).\\n3.5. Transcriptome analysis via GhCYP701A1 gene silencing\\nTo assess the change in gene expression levels of plants after silencing\\nthe GhCYP701A1 gene, transcriptome sequencing revealed a total of\\n15,962 differentially expressed genes, including 8376 upregulated and\\nFig. 1.(continued).\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n5'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 5, 'page_label': '6'}, page_content='Fig. 1.(continued).\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n6'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 6, 'page_label': '7'}, page_content='7586 downregulated genes (FDR corrections at the level of 0.05 and\\ngenes FPKM value /C21 1) (Fig. 3A). Among all the up-regulated genes,\\nthere were 2204 genes with more than double difference ratio. There are\\n2810 genes in the down-regulated genes (Fig. 3B).\\nTo determine the biological processes regulated by theGhCYP701A1\\ngene, all the differentially expressed genes obtained from the tran-\\nscriptome was subjected to GO enrichment analysis. GO analysis showed\\nthat the differentially expressed genes associated withGhCYP701A1 gene\\nsilencing were classiﬁed into three major categories: biological pro-\\ncesses, molecular functions, and cellular components (Fig. 4A). For bio-\\nlogical processes, they mainly included metabolic process, cellular\\nprocess, single-organism process, biological regulation, response to\\nstimulus, localization, cellular component organization or biogenesis,\\ndevelopmental process, signaling, multicellular organismal process,\\nreproductive process, multi-organism process, immune system process,\\ngrowth; reproduction, biological adhesion, rhythmic process, etc. Mo-\\nlecular functions covered catalytic activity, binding, transporter activity\\nnucleic acid binding transcription factor activity, molecular transducer\\nactivity, electron carrier activity, receptor activity, structural molecule\\nactivity, antioxidant activity, enzyme regulator activity, protein binding\\ntranscription factor activity, nutrient reservoir activity, guanyl-\\nnucleotide exchange factor activity, etc. Cellular components mainly\\nincluded cell part; organelle; membrane part; organelle part; macromo-\\nlecular complex; extracellular region; cell junction; membrane-enclosed\\nlumen; extracellular matrix; virion part; extracellular region part;\\nnucleoid, etc. Compared to TRV:00, the proportion of upregulated genes\\nin these GO enrichment pathways was signiﬁcantly higher than down-\\nregulated genes afterGhCYP701A1 gene silencing. However, in molec-\\nular functions, there were four pathways with downregulated genes\\nhigher than upregulated genes (nucleic acid binding transcription factor\\nactivity, molecular transducer activity, receptor activity, and antioxidant\\nactivity), while for biological processes, ﬁve pathways with down-\\nregulated genes were higher than upregulated genes (response to stim-\\nulus, localization, signaling, multi-organism process, and immune system\\nFig. 2. Plant phenotype and hormone content byGhCYP701A1 gene silencing.\\nA. Plant phenotype. Scale bars¼ 5 cm; B. The relative expression ofGhCYP701A1; C. The plant height; D. The content of GA3; E. The content of IAA; F. The content of\\nTZR; G. The content of JA; H. The content of ACC; I. The content of SA. The data are mean/C6 SE of three biological replicates. * and ** indicate statistical signiﬁcance\\nat the 0.05 and 0.01 probability level, respectively.\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n7'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 7, 'page_label': '8'}, page_content='process). Such the situation was not observed in cellular components\\n(Fig. 4B).\\nWe subsequently conducted KEGG enrichment analysis to gain a\\nbroader understanding for the biological changes that occurred in plants\\nafter silencing the GhCYP701A1 gene. The results showed that the\\ndifferentially expressed genes were mainly enriched in three major KEGG\\nclassiﬁcation pathways: metabolism; environmental information pro-\\ncessing; and organismal systems. Interestingly, organismal systems were\\nenriched in a metabolic pathway called plant-pathogen interaction,\\nwhich consisted of 113 differentially expressed genes. This pathway\\nincluded 32 up-regulated genes and 81 down-regulated genes. Addi-\\ntionally, two metabolic pathways related to environmental information\\nFig. 3. Statistical analysis of differently expressed genes (DEGs) between TRV:00 and TRV:GhCYP701A1.\\nA. Total number of up-regulated and down-regulated DEGs; B. Volcano distribution map of up-regulated and down-regulated DEGs.\\nFig. 4. Functional enrichment analysis of differently expressed genes (DEGs).\\nA. Enriched gene ontology (GO) terms for DEGs identiﬁed. The DEGs were classiﬁed based on GO annotation and were categorized into biological process (BP),\\nmolecular function (MF), and cellular component (CC); B. Bubble map of the top 20 KEGG enrichments for DEGs. The ordinate denotes the pathway name, theabscissa\\ndescribes the richness factor, each circle in the map represents a pathway, the size of a circle indicates the number of DEGs in the pathway, and the color of a circle\\ncorresponds to different p-value ranges; C. KO (KEGG Ontology) enrichment circle diagram. From the outside to the inside, theﬁrst circle represents the top 20\\nenrichment pathways, and the number outside the circle is the coordinate ruler of the number of genes; The second circle represents the number and Q value of\\nbackground genes in this pathway, and the more genes, the longer bar; The third circle represents the number of the DEGs in this pathway; The fourth circle represents\\nthe value of Rich Factor in each pathway.\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n8'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 8, 'page_label': '9'}, page_content='processing were identiﬁed: plant hormone signal transduction (234\\nDEGs, with 130 up-regulated genes and 104 down-regulated genes) and\\nABC transporters (10 DEGs, with six up-regulated genes and four down-\\nregulated genes). The remaining 17 pathways were categorized under\\nmetabolism. (Fig. 4C).\\nThe crucial role ofGhCYP701A1 gene in gibberellin synthesis was\\nhighlighted through KEGG enrichment analysis. Among the enriched\\nmetabolic pathways,‘plant hormone signal transduction’ stood out with\\n234 differentially expressed genes. These genes were predominantly\\ndistributed as follows: 96 involved in auxin signal transduction (70\\nupregulated, 26 downregulated), 43 in ABA signal transduction (10\\nupregulated, 33 downregulated), 23 in cytokinin signal transduction (12\\nupregulated, 11 downregulated), 20 in BR signal transduction (13\\nupregulated, 7 downregulated), 19 in JA signal transduction (14 upre-\\ngulated, 5 downregulated), 13 in GA signal transduction (all down-\\nregulated), and 12 in SA signal transduction (8 upregulated, 4\\ndownregulated). The silencing ofGhCYP701A1 gene impaired gibber-\\nellin synthesis, thereby leading to only 8 genes involving in the gibber-\\nellin signal transduction pathway compared to other hormone signal\\ntransduction pathways, with the least differentially expressed genes (3\\nupregulated, 5 downregulated). (Fig. 5, Table S1).\\nFig. 4.(continued).\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n9'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 9, 'page_label': '10'}, page_content='Fig. 4.(continued).\\nFig. 5. Pathway analysis of differentially expressed genes (DEGs) in hormone signaling.\\nThe neatly arranged blue circles form the cell membrane and gray dotted lines comprise the nucleus. The functionally annotated DEGs, all up-regulated genes are\\nrepresented by red boxes, all down-regulated genes are represented by green boxes, those containing both up- and down-regulated genes are represented by blue\\nboxes, and those without differentially expressed genes are represented by black boxes. A heat map of DEGs is placed next to the annotated gene.\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n10'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 10, 'page_label': '11'}, page_content='4. Discussion\\nCytochrome P450 enzymes, particularly the CYP701 family, have\\nbeen paid widespread attention in evolutionary studies, due to their\\npivotal roles in diverse biological processes across different organisms.\\nRecent research has provided valuable insights into the evolutionary\\nhistory, functional diversiﬁcation, and adaptive signiﬁcance of CYP701\\nenzymes. Phylogenetic analyses have revealed the ancient origin of the\\nCYP701 family, tracing back to early eukaryotic lineages. Comparative\\ngenomic studies have identiﬁed the ortholog ofCYP701 genes in various\\nplant species as well as in fungi and certain bacteria, suggesting their\\npresence in ancestral genomes and subsequent divergence during evo-\\nlution [9]. Evolutionary analysis ofCYP701A1 in various genomes of the\\nGossypium genus revealed thatGanoCYP701A1 from the B genome shows\\na closer evolutionary relationship to the outgroupGossypioides kirkiithan\\nCYP701A1 genes in other genomes. The remaining CYP701A1 genes\\nfrom different genomes can be categorized into three main branches. The\\nﬁrst clade comprises CYP701A1 gene from the allotetraploid at sub-\\ngenome, along with the genes from the A1, A2, and F1 genomes. The\\nsecond clade includesCYP701A1 genes from the C1, E1, G1, G2, and K\\ngenomes. Lastly, the third clade is composed ofCYP701A1 genes from\\nvarious D genomes and the allotetraploid Dt subgenome.\\nGibberellins (GAs) play pivotal roles in regulating various aspects of\\nplant growth and development, including seed germination, stem elon-\\ngation, andﬂower and fruit development. The inhibition or disruption of\\nGA biosynthesis, as observed in the case ofGhCYP701A1 gene silencing,\\nleads to signiﬁcant alterations in the hormonal balance within plants. The\\ninhibition of gibberellin (GA) biosynthesis or signaling pathways often\\nleads to signiﬁcant alterations in the levels and activities of other plant\\nhormones, thereby inﬂuencing various aspects of plant growth and\\ndevelopment. One notable consequence of gibberellin deﬁciency is the\\nupregulation of abscisic acid (ABA) biosynthesis and signaling pathways.\\nABA, known for its roles in stress responses and seed dormancy, tends to\\naccumulate in response to gibberellin depletion, leading to increased\\ntolerance to environmental stresses [34]. Additionally, ABA can inhibit\\nseed germination and promote root growth, thereby modulating plant\\ngrowth patterns under gibberellin-deﬁcient conditions [35]. Further-\\nmore, alterations in auxin levels and distribution patterns have been\\nobserved following gibberellin biosynthesis inhibition. Auxin, a key\\nregulator of plant growth and development, plays crucial roles in various\\nprocesses such as cell elongation, organ formation, and tropic responses\\n[36]. The crosstalk between gibberellins and auxin pathways is complex\\nand involves the regulation of auxin biosynthesis, transport, and\\nsignaling components [37]. When GhCYP701A1 was silenced inG. hir-\\nsutum ac. TM-1 using VIGS technology, transcriptome analysis showed\\nnotable alterations in the hormone signal transduction pathway. Subse-\\nquent measurement of hormone content in plants indicated a decrease in\\ngibberellin levels, along with signiﬁcant increases in auxin, cytokinin,\\nand jasmonic acid contents. Conversely, salicylic acid content decreased\\nsigniﬁcantly, while the ethylene synthesis precursor 1-aminocyclopropa-\\nne-1-carboxylic acid (ACC) content remained relatively unchangeable.\\n5. Conclusions\\nIn this study, genomic data from thirty Gossypium species were\\nanalyzed, which included nine allotetraploid genomes, 20 diploid ge-\\nnomes, and the Gossypioides kirkii genome used as an outgroup for\\nevolutionary analysis. The remaining CYP701A1 genes from different\\ngenomes were categorized into three main clades: the ﬁrst clade\\nencompassed CYP701A1 genes from the heterologous tetraploid at sub-\\ngenome, as well as those from the A1, A2, and F1 genomes; the second\\nclade includedCYP701A1 genes from the C1, E1, G1, G2, and K genomes;\\nand the third clade comprisedCYP701A1 genes from various D genomes\\nand the heterologous tetraploid Dt sub-genome. SilencingGhCYP701A1\\nin G. hirsutum ac. TM-1 using VIGS technology resulted in signiﬁcant\\nalterations in hormone levels. Gibberellin levels decreased, while auxin,\\ncytokinin, and jasmonic acid contents increased notably. Conversely,\\nsalicylic acid content decreased, while the precursor for ethylene syn-\\nthesis, 1-aminocyclopropane-1-carboxylic acid (ACC), remained rela-\\ntively stable.\\nFunding\\nThis program wasﬁnancially supported in part by grants from the\\nNational Science Foundation in China (31401429), the Natural Science\\nFoundation in Jiangsu Province (BK20140747).\\nCRediT authorship contribution statement\\nZhao Liang: Data curation, Funding acquisition, Writing– original\\ndraft. Di Jiachun:Data curation.Guo Qi:Data curation.Xu Zhenzhen:\\nFormal analysis.Zhao Jun:Formal analysis.Xu Peng:Data curation.Xu\\nJianwen: Data curation.Liu Jianguang:Data curation.Shen Xinlian:\\nConceptualization, Resources. Chen Xusheng: Writing – review &\\nediting.\\nDeclaration of competing interest\\nThe authors declare that they have no conﬂict of interest.\\nAcknowledgement\\nNone.\\nAppendix A. Supplementary data\\nSupplementary data to this article can be found online athttps://\\ndoi.org/10.1016/j.cropd.2024.100081.\\nReferences\\n[1] P. Achard, P. Genschik, Releasing the brakes of plant growth: how GAs shutdown\\nDELLA proteins, J. Exp. Bot. 60 (4) (2009) 1085–1092.\\n[2] P. Hedden, V. Sponsel, A century of gibberellin research, J. Plant Growth Regul. 34\\n(4) (2015) 740–760.\\n[3] J.M. Daviere, P. Achard, Gibberellin signaling in plants, Development 140 (6)\\n(2013) 1147–1151.\\n[4] J. MacMillan, Occurrence of gibberellins in vascular plants, fungi, and bacteria,\\nJ. Plant Growth Regul. 20 (4) (2002) 387–442.\\n[5] P. Hedden, A.L. Phillips, Gibberellin metabolism: new insights revealed by the\\ngenes, Trends Plant Sci. 5 (12) (2000) 523–530.\\n[6] T.P. Sun, Gibberellin Metabolism, Perception and Signaling Pathways in\\nArabidopsis, vol. 6, The Arabidopsis book/American Society of Plant Biologists,\\n2008 e0103.\\n[7] S. Ghosh, Triterpene structural diversiﬁcation by plant cytochrome P450 enzymes,\\nFront. Plant Sci. 8 (2017).\\n[8] Y.X. Li, K.F. Wei, Comparative functional genomics analysis of cytochrome P450\\ngene superfamily in wheat and maize, BMC Plant Biol. 20 (1) (2020).\\n[9] D. Nelson, D. Werck-Reichhart, A P450-centric view of plant evolution, Plant J. 66\\n(1) (2011) 194–211.\\n[10] S. Bak, F. Beisson, G. Bishop, B. Hamberger, R. Hofer, S. Paquette, D. Werck-\\nReichhart, Cytochromes P450, vol. 9, The Arabidopsis book/American Society of\\nPlant Biologists, 2011 e0144.\\n[11] D. Morrone, X.M. Chen, R.M. Coates, R.J. Peters, Characterization of the kaurene\\noxidase CYP701A3, a multifunctional cytochrome P450 from gibberellin\\nbiosynthesis, Biochem. J. 431 (2010) 337–344.\\n[12] M. Xu, Y. Lu, H. Yang, J. He, Z. Hu, X. Hu, M. Luan, L. Zhang, Y. Fan, L. Wang,\\nZmGRF, a GA regulatory factor from maize, promotesﬂowering and plant growth in\\nArabidopsis, Plant Mol. Biol. 87 (1–2) (2015) 157–167.\\n[13] N. Dhar, D.P.G. Short, B.E. Mamo, A.J. Corrion, C.M. Wai, A. Anchieta,\\nR. VanBuren, B. Day, H. Ajwa, K.V. Subbarao, S.J. Klosterman, Arabidopsis defense\\nmutant ndr1-1 displays accelerated development and earlyﬂowering mediated by\\nthe hormone gibberellic acid, Plant Sci. : an international journal of experimental\\nplant biology 285 (2019) 200–213.\\n[14] Y. Xu, Y. Wei, Z. Zhou, X. Cai, S.A. Boden, M.J. Umer, L.B. Safdar, Y. Liu, D. Jin,\\nY. Hou, et al., Widespread incomplete lineage sorting and introgression shaped\\nadaptive radiation in theGossypium genus, Plant Commun 5 (2) (2024) 100728.\\n[15] M.J. Wang, J.Y. Li, Z.Y. Qi, Y.X. Long, L.L. Pei, X.H. Huang, C.E. Grover, X.M. Du,\\nC.J. Xia, P.C. Wang, et al., Genomic innovation and regulatory rewiring during\\nevolution of the cotton genusGossypium, Nat. Genet. 54 (12) (2022) 1959–1971.\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n11'),\n",
       " Document(metadata={'producer': 'Acrobat Distiller 8.1.0 (Windows)', 'creator': 'Elsevier', 'creationdate': '2025-02-24T19:46:21+05:30', 'crossmarkdomains[2]': 'elsevier.com', 'crossmarkmajorversiondate': '2010-04-23', 'subject': 'Crop Design, 4 (2025) 100081. doi:10.1016/j.cropd.2024.100081', 'author': 'Zhao Liang', 'elsevierwebpdfspecifications': '7.0.1', 'crossmarkdomainexclusive': 'true', 'robots': 'noindex', 'moddate': '2025-02-24T19:48:16+05:30', 'doi': '10.1016/j.cropd.2024.100081', 'crossmarkdomains[1]': 'sciencedirect.com', 'title': 'Analysis of CYP701A1 genes in gossypium species and functional characterization through gene silencing', 'source': '1-s2.0-S2772899424000302-main.pdf', 'total_pages': 12, 'page': 11, 'page_label': '12'}, page_content='[16] C. Shen, N.A. Wang, D. Zhu, P.C. Wang, M.J. Wang, T.W. Wen, Y. Le, M. Wu, T. Yao,\\nX.L. Zhang, Z.X. Lin,Gossypium tomentosum genome and interspeciﬁc ultra-dense\\ngenetic maps reveal genomic structures, recombination landscape andﬂowering\\ndepression in cotton, Genomics 113 (4) (2021) 1999–2009.\\n[17] Y. Hu, J. Chen, L. Fang, Z. Zhang, W. Ma, Y. Niu, L. Ju, J. Deng, T. Zhao, J. Lian, et\\nal., Gossypium barbadenseand Gossypium hirsutumgenomes provide insights into the\\norigin and evolution of allotetraploid cotton, Nat. Genet. 51 (4) (2019) 739–748.\\n[18] Z.J. Chen, A. Sreedasyam, A. Ando, Q.X. Song, L.M. De Santiago, A.M. Hulse-Kemp,\\nM.Q. Ding, W.X. Ye, R.C. Kirkbride, J. Jenkins, et al., Genomic diversiﬁcations of\\nﬁve Gossypium allopolyploid species and their impact on cotton improvement, Nat.\\nGenet. 52 (5) (2020) 525–533.\\n[19] R.H. Peng, Y.C. Xu, S.L. Tian, T. Unver, Z. Liu, Z.L. Zhou, X.Y. Cai, K.B. Wang,\\nY.Y. Wei, Y.L. Liu, et al., Evolutionary divergence of duplicated genomes in newly\\ndescribed allotetraploid cottons, Proc. Natl. Acad. Sci. U.S.A. 119 (39) (2022).\\n[20] G. Huang, Z.G. Wu, R.G. Percy, M.Z. Bai, Y. Li, J.E. Frelichowski, J. Hu, K. Wang,\\nJ.Z. Yu, Y.X. Zhu, Genome sequence ofGossypium herbaceumand genome updates of\\nGossypium arboreumand Gossypium hirsutumprovide insights into cotton A-genome\\nevolution, Nat. Genet. 52 (5) (2020) 516–524.\\n[21] Z.Z. Xu, J.D. Chen, S. Meng, P. Xu, C.J. Zhai, F. Huang, Q. Guo, L. Zhao, Y.G. Quan,\\nY.X. Shangguan, et al., Genome sequence ofGossypium anomalum facilitates\\ninterspeciﬁc introgression breeding, Plant Commun 3 (5) (2022) 100350.\\n[22] K. Sheng, Y. Sun, M. Liu, Y.F. Cao, Y.F. Han, C. Li, U. Muhammad, M.K. Daud,\\nW.R. Wang, H.Z. Li, et al., A reference-grade genome assembly forGossypium bickii\\nand insights into its genome evolution and formation of pigment glands and\\ngossypol, Plant Commun 4 (1) (2023) 100421.\\n[23] Y. Cai, X. Cai, Q. Wang, P. Wang, Y. Zhang, C. Cai, Y. Xu, K. Wang, Z. Zhou,\\nC. Wang, et al., Genome sequencing of the Australian wild diploid species\\nGossypium australe highlights disease resistance and delayed gland morphogenesis,\\nPlant Biotechnol. J. 18 (3) (2020) 814–828.\\n[24] Z.E. Yang, X.Y. Ge, W.N. Li, Y.Y. Jin, L.S. Liu, W. Hu, F.Y. Liu, Y.L. Chen, S.L. Peng,\\nF.G. Li, Cotton D genome assemblies built with long-read data unveil mechanisms of\\ncentromere evolution and stress tolerance divergence, BMC Biol. 19 (1) (2021).\\n[25] C.E. Grover, M.A. Arick, A. Thrash, J.L. Conover, W.S. Sanders, D.G. Peterson,\\nJ.E. Frelichowski, J.A. Schefﬂer, B.E. Schefﬂer, J.F. Wendel, Insights into the\\nevolution of the new world diploid cottons (Gossypium, subgenus Houzingenia)\\nbased on genome sequencing, Genome Biology and Evolution 11 (1) (2018) 53–71.\\n[26] A.H. Paterson, J.F. Wendel, H. Gundlach, H. Guo, J. Jenkins, D. Jin, D. Llewellyn,\\nK.C. Showmaker, S. Shu, J. Udall, et al., Repeated polyploidization ofGossypium\\ngenomes and the evolution of spinnable cottonﬁbres, Nature 492 (7429) (2012)\\n423–427.\\n[27] J.A. Udall, E. Long, C. Hanson, D. Yuan, T. Ramaraj, J.L. Conover, L. Gong,\\nM.A. Arick, C.E. Grover, D.G. Peterson, J.F. Wendel,De novo genome sequence\\nassemblies of Gossypium raimondii and Gossypium turneri,G 3\\nGenesjGenomesjGenetics 9 (10) (2019) 3079–3085.\\n[28] J.A. Udall, E. Long, T. Ramaraj, J.L. Conover, D.J. Yuan, C.E. Grover, L. Gong,\\nM.A. Arick, R.E. Masonbrink, D.G. Peterson, J.F. Wendel, The genome sequence of\\nGossypioides kirkii illustrates a descending dysploidy in plants, Front. Plant Sci. 10\\n(2019).\\n[29] B. Hu, J.P. Jin, A.Y. Guo, H. Zhang, J.C. Luo, G. Gao, Gsds 2.0: an upgraded gene\\nfeature visualization server, Bioinformatics 31 (8) (2015) 1296–1297.\\n[30] S. Kumar, G. Stecher, M. Li, C. Knyaz, K. Tamura, X. Mega, Molecular evolutionary\\ngenetics analysis across computing platforms, Mol. Biol. Evol. 35 (6) (2018)\\n1547–1549.\\n[31] M. Lescot, P. D/C19ehais, G. Thijs, K. Marchal, Y. Moreau, Y. Van de Peer, P. Rouz/C19e,\\nS. Rombauts, PlantCARE, a database of plantcis-acting regulatory elements and a\\nportal to tools for analysis of promoter sequences, Nucleic Acids Res. 30 (1) (2002)\\n325–327.\\n[32] K.J. Livak, T.D. Schmittgen, Analysis of relative gene expression data using real-\\ntime quantitative PCR and the 2-ΔΔCT Method, Methods 25 (4) (2001) 402–408.\\n[33] D. Ma, Y. Hu, C. Yang, B. Liu, L. Fang, Q. Wan, W. Liang, G. Mei, L. Wang, H. Wang,\\net al., Genetic basis for glandular trichome formation in cotton, Nat. Commun. 7\\n(2016) 10456.\\n[34] M. Seo, T. Koshiba, Complex regulation of ABA biosynthesis in plants, Trends Plant\\nSci. 7 (1) (2002) 41–48.\\n[35] R.R. Finkelstein, S.S.L. Gampala, C.D. Rock, Abscisic acid signaling in seeds and\\nseedlings, Plant Cell 14 (2002) S15–S45.\\n[36] R. Swarup, R. Bhosale, Developmental roles of AUX1/LAX auxin inﬂux carriers in\\nplants, Front. Plant Sci. 10 (2019).\\n[37] X.D. Fu, N.P. Harberd, Auxin promotesArabidopsis root growth by modulating\\ngibberellin response, Nature 421 (6924) (2003) 740–743.\\nZ. Liang et al. Crop Design 4 (2025) 100081\\n12')]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# PDF Loader\n",
    "from langchain_community.document_loaders import PyPDFLoader\n",
    "loader = PyPDFLoader('1-s2.0-S2772899424000302-main.pdf')\n",
    "data = loader.load()\n",
    "data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "9bf8c10a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    }
   ],
   "source": [
    "# Creating chunks from documents we loaded from the PDF\n",
    "from langchain_text_splitters import RecursiveCharacterTextSplitter\n",
    "\n",
    "text_splitter = RecursiveCharacterTextSplitter(\n",
    "    chunk_size=1000,\n",
    "    chunk_overlap=200,\n",
    ")\n",
    "\n",
    "docs = text_splitter.split_documents(data)\n",
    "print(len(docs))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8c868484",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Creating embeddings and storing them in a vector store\n",
    "\n",
    "# We can also use other embedding models like OpenAIEmbeddings, HuggingFaceEmbeddings, etc.\n",
    "from langchain_ollama import OllamaEmbeddings\n",
    "# We can also use other vector stores like Chroma, Pinecone, etc.\n",
    "from langchain_community.vectorstores import FAISS\n",
    "\n",
    "embedding_model = OllamaEmbeddings(model=\"nomic-embed-text\")\n",
    "vectorstore = FAISS.from_documents(docs, embedding_model)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "519737f0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# query = \"Who funded for the research?\"\n",
    "# result = vectorstore.similarity_search(query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "c7b40cfd",
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_ollama import OllamaLLM\n",
    "llm = OllamaLLM(model=\"llama3\", temperature=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "8365fcef",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Design Prompt\n",
    "from langchain_core.prompts import ChatPromptTemplate\n",
    "\n",
    "prompt = ChatPromptTemplate.from_template(\n",
    "    \"\"\"Only use the following context to answer the question, if the context is not sufficient, say 'I don't know'.\n",
    "    context: {context}\n",
    "    question: {question}\n",
    "    \"\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "c7220c4a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "VectorStoreRetriever(tags=['FAISS', 'OllamaEmbeddings'], vectorstore=<langchain_community.vectorstores.faiss.FAISS object at 0x000001413602E2A0>, search_kwargs={'k': 4})"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Creating the chain to combine the retriever and the LLM\n",
    "\n",
    "retriever = vectorstore.as_retriever(search_kwargs={\"k\": 4})\n",
    "retriever"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "f7410fe1",
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_core.runnables import RunnablePassthrough\n",
    "\n",
    "rag_chain = (\n",
    "    {\n",
    "        \"context\": retriever,\n",
    "        \"question\": RunnablePassthrough()\n",
    "    }\n",
    "    | prompt\n",
    "    | llm\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "476e372d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The research was financially supported in part by grants from the National Science Foundation in China (31401429) and the Natural Science Foundation in Jiangsu Province (BK20140747).\n"
     ]
    }
   ],
   "source": [
    "result = rag_chain.invoke(\"Who funded for the research?\")\n",
    "print(result)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
